### PR TITLE
Allow building smashbox with any client branch

### PR DESCRIPTION
**Pull request by mrow4a**
@tboerger

### CODE REVIEW

File: rootfs/usr/local/bin/smash-wrapper [hunk 1]
```diff
@@ -8,18 +8,19 @@ declare -x SMASHBOX_USERNAME
...
+
+TEST_FOLDER=/smashbox/lib/jenkins
 
 [[ -z "${SMASHBOX_TIMEOUT}" ]] && SMASHBOX_TIMEOUT="3600"
 [[ -z "${SMASHBOX_WAIT}" ]] && SMASHBOX_WAIT="false"
-[[ -z "${SMASHBOX_TEST_SET}" ]] && SMASHBOX_TEST_SET="-a"
 [[ -z "${SMASHBOX_CHECK_URL}" ]] && SMASHBOX_CHECK_URL="${SMASHBOX_URL}"
 [[ -z "${SMASHBOX_CHECK_STATUS}" ]] && SMASHBOX_CHECK_STATUS="200"
 [[ -z "${SMASHBOX_SSL_ENABLED}" ]] && SMASHBOX_SSL_ENABLED="false"
+[[ -z "${SMASHBOX_CLIENT_VERSION}" ]] && SMASHBOX_CLIENT_VERSION="master"
```
Comment (mrow4a) [inline rootfs/usr/local/bin/smash-wrapper]: @tboerger @guruz this possibly allows you to run custom branch and test it

**Comment by mrow4a** [comment]
@tboerger please create branch build, so this PR will be directed there

**Comment by tboerger** [comment]
I have created that a little bit different now: https://github.com/owncloud-docker/smashbox/commit/bc9921e4add96b652c75c904d141068b5531eadf

But thank you for the initial effort.

**Comment by mrow4a** [comment]
Yee your version is much cleaner :> Thanks for the effort of commenting the code so I actually could learn how to do things like that, really good code!

**Pull request closed without merging**